pkg/tinyusb/doc.txt

```diff
@@ -51,20 +51,26 @@
  * }
  * ```
  *
- * Create a file `tusb_config.h` in your application directory which includes
- * at least the file `tinyusb_config.h` from the tinyUSB package. This file is
- * required by the tinyUSB stack and can be used to override the default
- * configuration defined in `tinyusb_config.h`.
+ * Create a file `tinyusb_app_config.h` in your application directory.
+ * This file is can be used to override the default configuration defined
+ * in `tusb_config.h`.
  * ```c
  * #define CONFIG_TUSBD_CDC_NUMOF   2
- *
- * #include "tinyusb_config.h"
  * ```
  *
+ * By default, the number of `CFG_TUD_*` device class and `CFG_TUH_*`
+ * host class interfaces are defined to 1 if the corresponding `tinyusb_class_*`
+ * and `tinyusb_device`/`tinyusb_host` module are enabled, and 0 otherwise.
```

Comment:
```suggestion
 * By default, the number of `CONFIG_TUSBD_*_NUMOF` device class and
 * `CONFIG_TUSBH_*_NUMOF` host class interfaces are defined to 1 if the
 * corresponding `tinyusb_class_*` and `tinyusb_device`/`tinyusb_host`
 * module are enabled, and 0 otherwise.
```
Hm, this change request was somehow lost.